experimental/FTheoryTools/src/AbstractFTheoryModels/methods.jl

```diff
@@ -72,10 +72,7 @@ julia> x1, x2, x3, x4, x, y, z = gens(cox_ring(ambient_space(t)))
 7-element Vector{MPolyDecRingElem{QQFieldElem, QQMPolyRingElem}}:
  x1
  x2
- x3
- x4
- x
- y
+ ⋮
```

Comment:
These changes are unfortunately a bottleneck of  `doctest_fix`, which makes use of your terminal size for abbreviating lists. The doctests will fail with this, so I am afraid you have to revert this change (and the one a bit further down in this file)